clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -610,6 +628,18 @@ public class ConsumerConfig extends AbstractConfig {
                                         DEFAULT_ALLOW_AUTO_CREATE_TOPICS,
                                         Importance.MEDIUM,
                                         ALLOW_AUTO_CREATE_TOPICS_DOC)
+                                .define(GROUP_PROTOCOL_CONFIG,
+                                    Type.STRING,
+                                    DEFAULT_GROUP_PROTOCOL,
+                                    ConfigDef.CaseInsensitiveValidString
+                                        .in(Utils.enumOptions(GroupProtocol.class)),
+                                    Importance.HIGH,
+                                    GROUP_PROTOCOL_DOC)
+                                .define(REMOTE_ASSIGNOR_CONFIG,
+                                    Type.STRING,
+                                    DEFAULT_REMOTE_ASSIGNOR,
+                                    Importance.MEDIUM,
+                                    REMOTE_ASSIGNOR_DOC)
```

Comment:
nit: It looks like the indentation is off by four spaces compared to the rest of the code.

Reply:
You are right! I kept it that way because I was trying to avoid editing a few hundred lines of code!

Reply:
@philipnee It would be better to update the new code to follow the existing style. This is what we usually do…